swfiles/@spinw/intmatrix.m

```diff
@@ -62,31 +62,38 @@
 % ### Output Arguments
 % 
 % `SS`
-% : structure with fields `iso`, `ani`, `dm`, `gen`, `bq`, `dip` and
-%   `all`. It describes the bonds between spins. Every field is a matrix,
-%               where every column is a coupling between two spins. The
-%               first 3 rows contain the unit cell translation vector
-%               between the interacting spins, the 4th and 5th rows contain
-%               the indices of the two interacting spins in the
-%               [spinw.matom] list. The following rows contains the
-%               strength of the interaction. For isotropic exchange it is a
-%               single number, for DM interaction it is a column vector
-%               `[DMx; DMy; DMz]`, for anisotropic interaction `[Jxx; Jyy;
-%               Jzz]` and for general interaction `[Jxx; Jxy; Jxz; Jyx; Jyy;
-%               Jyz; Jzx; Jzy; Jzz]` and for biquadratic exchange it is also
-%               a single number. For example:
-%   ```
-%   SS.iso = [dl_a; dl_b; dl_c; matom1; matom2; Jval]
-%   ```
+% : structure where every field is a matrix. Every column is a coupling 
+%   between two spins. The first 3 rows contain the unit cell translation 
+%   vector between the interacting spins, the 4th and 5th rows contain
+%   the indices of the two interacting spins in the [spinw.matom] list. 
+%   Subsequent rows in the matrix depend on the field
+%   SS will always have the following fields
+%   * `all`
+%   * `dip`
+%   Subsequent rows in these matrices are the elements of the 3 x 3 
+%   exchange matrix `[Jxx; Jxy; Jxz; Jyx; Jyy; Jyz; Jzx; Jzy; Jzz]`
+%   and the final row indicates whether the coupling is
+%   quadratic (0) or biquadratic (1). The `dip` field contains the dipolar
+%   interactions only that are not added to `SS.all.
```

Comment:
I think `quadratic (0)` here should be `bilinear (0)`. The "bi" refers to pair-interactions between spins; `Ji.Jj` is a linear pair interaction whilst `(Ji.Jj)^2` is a quadratic interaction.

Also, you're missing closing &#96; after `SS.all`.